Source/WebCore/Modules/mediasource/SourceBuffer.h

```diff
@@ -107,7 +107,7 @@ class SourceBuffer
     void removedFromMediaSource();
     void seekToTime(const MediaTime&);
 
-    bool canPlayThroughRange(PlatformTimeRanges&);
+    bool canPlayThroughRange(const PlatformTimeRanges&);
```

Comment:
Now that you add a `const`. I think it would make the whole function `const`, but I don't know if it can create other build issues.

Reply:
It cannot be a `const` function because it calls `monitorBufferingRate()`, which mutates state.